### PR TITLE
Audio

### PR DESCRIPTION
**Pull request by alexgoff**

### CODE REVIEW

File: src/components/HUD/score.jsx [hunk 1]
```diff
@@ -51,14 +52,12 @@ const Score = styled.div`
 const HUDScore = ({ score }) => {
   const { ref, width, height } = useResizeObserver();
 
-  const values = Object.values(score);
-
-  const sum = values.reduce((accumulator, value) => accumulator + value, 0);
+  const scoreSum = sum(Object.values(score));
```
Comment (alexgoff) [inline src/components/HUD/score.jsx:55]: added this `sum` util for figuring out if the score changed, used it other places too

File: src/constants/timeline.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ export const DAY_TRANSITION_DURATION: number = 2000;
 export const FINISH_SCREEN_DURATION: number = 10000;
 
 // Starts
-export const TIMER_START: number = 1000;
+export const TIMER_START: number = 3500;
```
Comment (alexgoff) [inline src/constants/timeline.ts:12]: extend start time to play countdown

File: src/entities/audio.ts [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import { Howl } from 'howler';
+
+export default {
```
Comment (alexgoff) [inline src/entities/audio.ts:3]: these will be replaced with inlined versions and need fallback `.webm` formats

File: src/systems/audio.ts [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      e.type === 'gameStart' ||
+      e.type === 'timeStart' ||
+      e.type === 'timeEnd' ||
+      e.type === 'cameraExposing' ||
+      e.type === 'scoreUpdate' ||
+      e.type === 'cameraExposureEnd' ||
+      e.type === 'cameraMoving'
+  );
+
+  if (event) {
```
Comment (alexgoff) [inline src/systems/audio.ts:15]: simple event handler to play corresponding audio

File: src/systems/camera.ts [hunk 5]
```diff
@@ -129,31 +134,24 @@ const onCameraExposing: GameSystem = (entities, { events, time, dispatch }) => {
     const exposureElapsed = current - exposureStartTime;
 
     if (isFirstExposure) {
+      state.lastScore = sum(Object.values(score));
```
Comment (alexgoff) [inline src/systems/camera.ts:137]: at the start of exposure log the total score in the game state

File: src/systems/camera.ts [hunk 6]
```diff
@@ -129,31 +134,24 @@ const onCameraExposing: GameSystem = (entities, { events, time, dispatch }) => {
...
-      };
     } else {
-      dispatch({ type: 'cameraExposureEnd' });
-      return {
-        ...entities,
-        camera: {
-          ...camera,
-          exposureRemaining: null,
-          exposureStartTime: null,
+      camera.exposureRemaining = null;
```
Comment (alexgoff) [inline src/systems/camera.ts:147]: move to state mutation so that spreading does not create new `Howl` instances and reload the audio files

**Review by blnkt — APPROVED** [review]
I think I have no notes.  I agree with all the little things you’re calling out in your inline comments.  The Team and the project (and I) are going to be so impressed with this.  Is feeling very polished.

**Pull request merged**